### PR TITLE
Lower expiry threshold to 2 hours

### PR DESCRIPTION
**Pull request by Jibola**

### CODE REVIEW

File: astrolabe/atlas_runner.py [hunk 1]
```diff
@@ -458,8 +453,8 @@ def __init__(
         self.persist_clusters = persist_clusters
         self.no_create = no_create
         self.workload_startup_time = workload_startup_time
-        # Hardcoded to 12 hours, can be configurable in the future
-        self.project_expiration_threshold_seconds = 12 * 60 * 60
+        # Hardcoded to 2 hours, can be configurable in the future
+        self.project_expiration_threshold_seconds = 2 * 60 * 60
```
Comment (Jibola) [inline astrolabe/atlas_runner.py:457]: These are the only two lines that matter. The rest was autoformat

**Review by blink1073 — APPROVED** [review]
LGTM!

File: astrolabe/atlas_runner.py [hunk 2]
```diff
@@ -570,8 +559,9 @@ def clean_old_projects(self, org_id):
                         else:
                             raise
                 else:
+                    LOGGER.info("Skipping deleting project %r, id: %r", project.name, project.id)
                     LOGGER.info(
-                        "Skipping deleting project %r, id: %r", project.name, project.id
+                        f"{project_timestamp=}, {current_timestamp=}, {self.project_expiration_threshold_seconds=}"  # noqa: G004
```
Comment (Jibola) [inline astrolabe/atlas_runner.py:564]: @blink1073, @matthewdale  this actually snaked in, I wonder if we should still keep it as a valuable debugging datapoint

Comment (blink1073) [reply astrolabe/atlas_runner.py:564]: Yes I think so

**Pull request merged**